### DISCUSSION TITLE
fandom.com modal

### DISCUSSION THREAD

**Discussion by stephenhawk8054** [uBlockOrigin/uAssets#18203] [General]
From this issue: `https://www.reddit.com/r/uBlockOrigin/comments/13p72m4/`

URL: `https://battle-cats.fandom.com/wiki/Cat_Dictionary`

Needs to access via android (or simulated android on PC).

The modal is caused by this script: `https://static.wikia.nocookie.net/silversurfer/prod/latest/ageGateUnit-pathfinder.js`. I'm not sure if this counts as age check modal to put in annoyance list or not?

**Comment by mapx-** [comment]
definitely as annoyance

**Reply by stephenhawk8054** [reply]
I mean our list (and fanboy annoyance list) does not address age check (like `I'm over 18`) sites, but I'm not sure if this site is counted as one of them or it's just a way to collect data?

**Reply by mapx-** [reply]
`I'm over 18` is legit imo, this 1 instead could be seen as annoyance, but let's see other opinions, @okiehsch what to do you think about ?

**Reply by okiehsch** [reply]
Agreed, it's not an age check, it's an option to change the sites visual appearance and I would certainly consider it to be an annoyance.

**Reply by stephenhawk8054** [reply]
Added: https://github.com/uBlockOrigin/uAssets/commit/671d8e31645d2aa6f700a66c8892db1a0a141c1a

**Comment by Yuki2718** [comment]
I don't see any modal either on real or simulated Android.

**Reply by MasterKia** [reply]
Same.

**Reply by Yuki2718** [reply]
Maybe only if accessed from Europe? I'm now running PyFunceble so don't wanna use VPN (can interrupt).

**Reply by stephenhawk8054** [reply]
Looks like it only appears if you change the theme to Dark theme (Hamburger menu at top right > Dark theme)

**Reply by Yuki2718** [reply]
Still nothing.
